py/requirements.txt

```diff
@@ -2,7 +2,7 @@ numpy
 packaging
 pybind11==2.6.2
 --extra-index-url https://download.pytorch.org/whl/nightly/cu124
-torch>=2.5.0.dev,<2.6.0
+torch>=2.6.0.dev,<2.7.0
 torchvision>=0.20.0.dev,<0.21.0
```

Comment:
should torchvision be bumped as well ?

Reply:
torchvision is still 0.20.0.dev for torch2.6.0.dev